### DISCUSSION TITLE
reference/docs/mysql

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#489] [Q&A]
# reference/docs/mysql

http://bbs.laoleng.vip/reference/docs/mysql.html

**Comment by bobuting** [comment]
111

**Comment by tianguoao** [comment]
2 嗯 231

**Comment by SheriocCode** [comment]
补充：
查看最近一次执行sql错误日志
SHOW ERRORS;

**Comment by wx40217** [comment]
补充：
GET_JSON_OBJECT(json_doc, path) 从JSON字符串中提取特定数据
